src/components/loader-star/loader-star.test.tsx

```diff
@@ -40,7 +40,7 @@ test("should return null and render nothing when useMediaQuery returns undefined
 });
 
 test("component should have a visually hidden label for assistive technologies by default", () => {
-  mockUseMediaQuery.mockReturnValueOnce(true);
+  mockUseMediaQuery.mockReturnValueOnce(false);
```

Comment:
comment: the hidden label will only render when motion is allowed so this should be `true`